main/train_classifier_g.py

```diff
@@ -56,33 +56,41 @@ def train_model(training_data, model_name="svm", pretrain=False, what_data="issu
   elif model_name == "rf":
     s.set_model_function(classifiers.random_forest_model)
     # RF params
-    s.add_parameter("n_estimators",
-                    [int(x) for x in np.linspace(start=200, stop=2000, num=10)])
-    s.add_parameter("max_features", ["auto", "sqrt"])
-    s.add_parameter("max_depth", [int(x) for x in np.linspace(10, 110, num=11)])
+    s.set_parameters({
+        "n_estimators": [int(x) for x in np.linspace(start=10, stop=100, num=10)],
```

Comment:
it's generally best to separate pull requests into small pieces. for instance, I'd put this parameter tweaking into its own one. if this set_parameters refactoring is more than just these 4 lines, I'd also have a separate refactoring PR

Reply:
I won't insist on this now, since we're in a hurry

Reply:
Got it~